internal/pkg/cli/errors.go

```diff
@@ -130,3 +131,23 @@ func (e *errNoInfrastructureChanges) Error() string {
 func (e *errNoInfrastructureChanges) ExitCode() int {
 	return 0
 }
+
+type errBucketEmptyingFailed struct {
+	failedBuckets []string
+	bucketErrors  []error
+}
+
+func (e *errBucketEmptyingFailed) Error() string {
```

Comment:
Ah, forgot we had [`errors.Join`](https://pkg.go.dev/errors#Join) - new in go 1.20! We should consider using that instead. Example usage: https://github.com/aws/copilot-cli/blob/a9f96c983b25f85925714393c52dfc0b8466565c/internal/pkg/aws/s3/s3.go#L133